### ISSUE TITLE
AttributeError: module 'aiohttp' has no attribute 'ws_connect'

### ISSUE THREAD

**Issue by nkgsv** [aio-libs/aiohttp#4783]
When running [client_ws.py](https://github.com/aio-libs/aiohttp/blob/master/examples/client_ws.py) I get the following error:

```
Traceback (most recent call last):
  File "./client_ws.py", line 46, in start_client
    async with aiohttp.ws_connect(url, autoclose=False, autoping=False) as ws:
AttributeError: module 'aiohttp' has no attribute 'ws_connect'
```

https://github.com/aio-libs/aiohttp/blob/385b03ef21415d062886e1caab74eb5b93fdb887/examples/client_ws.py#L46

Shall I use `aiohttp.ClientSession().ws_connect` instead of `aiohttp.ws_connect`?

**Comment by webknjaz** [comment]
Yes. Those examples are pretty old so some things may be outdated. Feel free to send a PR to correct them.

Ref: https://docs.aiohttp.org/en/latest/client_quickstart.html#aiohttp-client-websockets

**Comment by asvetlov** [comment]
Fixed by 97de69f1 I hope
